stories/topics/proc-aap-aws-backup-and-recovery.adoc

```diff
@@ -7,113 +7,121 @@ Follow the steps below to ensure a smooth backup and recovery process.
+
+.. Run the command to run the playbook.
++
+[source,bash]
+----
+$ docker run --rm --env PLATFORM=AWS \
+-v <absolute path of extra vars dir>:/extra_vars:ro \
+-e AWS_ACCESS_KEY_ID=<Your AWS Access Key ID>
+-e AWS_SECRET_ACCESS_KEY=<Your AWS Secret Access Key>
+<image path still needs to be determined>:latest \
+redhat.ansible_on_clouds.aws_backup_stack \
+-e @/extra_vars/extra_vars.yaml
+----
+
+.. Note the output of the playbook
++
+The playbook will return something that looks like this
++
+[source, bash]
+----
+{
+    "efs_restore_points": [
+        {
+            "CreationDate": "2023-02-28T21:54:07.515000+00:00",
+            "RecoveryPointArn": "arn:aws:backup:us-east-1:080914983077:recovery-point:967be465-a43e-432a-b536-8b16e6b0452d"
+        },
+        {
+            "CreationDate": "2023-02-28T16:08:08.905000+00:00",
+            "RecoveryPointArn": "arn:aws:backup:us-east-1:080914983077:recovery-point:d2058e3a-dca6-4a18-b175-5ef624f513a7"
+        }
+    ],
+    "rds_db_snapshot_arn": "arn:aws:rds:us-east-1:080914983077:snapshot:test-bnr-ops-container-rds169785b9-orm2iuzlfqem-snap-2023-02-28"
+}
+----
++
+Make a note of the *rds_db_snapshot_arn* and the latest *RecoveryPointArn*. These will be needed for the restore playbook
++
+
+
+== Recovery Process
 
 .Procedure
-. In the AWS Console, navigate to menu:CloudFormation[].
-. Click on btn:[Create Stack] near the top right of the screen.
-. Select *With new resources (standard)* in the dropdown.
-. In the *Specify template* section under *Template source* click btn:[Upload a template file] and select the template you have modified.
-. Click btn:[Next] to go to the next step
-. Enter a stack name and select an AWSKeyPair to use for the deployment.
-. Click btn:[Next] to go to the *Configure stack options* page.
-. Optional: You can configure stack options and click btn:[Next] to review the stack
-. Click btn:[Submit] to create your new deployment using an RDS Snapshot and restored EFS.
-
-
-You can now log in succesfully to {PlatformName} {ControllerName} and {HubName} using your old deployment credentials. 
+. Gather the following input variables to create an `extra_vars.yaml` file.
+.. aws_foundation_stack_name: The name of your existing deployment
+.. aws_restored_stack_name: The name of your new deployment
+.. aws_region: The region the stack was deployed in
+.. aws_backup_vault_name: The name of the backup vault your EFS backup is stored in
+.. aws_rds_db_snapshot_arn: The ARN of the RDS snapshot you want to use for restore. Can be found from the backup playbook output as *rds_db_snapshot_arn*. 
+[NOTE]
+If you have a specific RDS snapshot you want to use, you will have to manually find its ARN in the AWS console. 
+.. aws_backup_restore_point_arn: The ARN of the EFS recovery point you want to use for restore. Can be found from the backup playbook output as *RecoveryPointArn*. 
```

Comment:
The ARN of the recovery point you want to use for restore, which can be found...